epay3.Web.Api.Tests/Processor13/TransactionsFixture.cs

```diff
@@ -0,0 +1,230 @@
+﻿using epay3.Web.Api.Sdk.Api;
+using epay3.Web.Api.Sdk.Model;
+using epay3.Web.Api.Tests.TestData;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Linq;
+using System.Net;
+
+namespace epay3.Web.Api.Tests.Processor13
+{
```

Comment:
This entire fixture fails against sandbox. Do I need to do some special configuration?